source/guides/clear/telemetrics.rst

```diff
@@ -163,24 +163,27 @@ Enable or disable telemetry
    During the initial installation of |CL|, you are requested to join the
    stability enhancement program and allow |CL| to collect anonymous reports to
    improve system stability. If you choose not to join this program, then the
-   telemetry software bundle is not added to your system. Choosing to join will
-   automatically enable telemetry on your system after installation is
-   complete.
+   telemetry software bundle is not added to your system. If you do choose to
+   join the program, the installer will automatically enable telemetry on your
+   system by installing the telemetrics bundle, creating the file
+   :file:`/etc/telemetrics/opt-in`, and enabling the telemtrics systemd
+   services to run after installation is complete and the system is restarted.
 
 #. Enabling after install:
 
-   To start telemetry on your system, run the following command:
+   To install telemetry on your system, run the following commands:
 
    .. code-block:: bash
 
+      sudo swupd bundle-add telemetrics
       sudo telemctl opt-in
       sudo telemctl start
 
-   This enables and starts the :command:`telemprobd` and :command:`telempostd`
-   daemons. Your system will begin to send telemetry data to the server defined
-   in the file :file:`/etc/telemetrics/telemetrics.conf`. If this file does not
-   exist, the :command:`telemprobd` and :command:`telempostd` daemons will use
-   the file :file:`/usr/share/defaults/telemetrics/telemetrics.conf`.
+   This installs the necassary software, enables telemetry by creating the file
+   :file:`/etc/telemetrics/opt-in`, and starts the :command:`telemprobd` and
+   :command:`telempostd` daemons. Your system will begin to send telemetry data to the server defined in the file :file:`/etc/telemetrics/telemetrics.conf`. If this file does not exist, the :command:`telemprobd` and
+   :command:`telempostd` daemons will use the file
```

Comment:
OK we messed up here. This has not being true (daemons will use the file...) since 2.2.1 was released, telemetrics now uses defaults and the default is `clr.telemetry.intel.com` if you wish to change it you add the file at `/etc/telemetrics/telemetrics.conf` with the keys of the values that you wish to change. `/usr/share/defaults/telemetrics/telemetrics.conf` still exists, however is only for reference.